### PR TITLE
Fix error parsing to idt_int

### PR DESCRIPTION
**Pull request by jostar-yang**
**- What I did**
Fix error parsing to idt_int
**- How I did it**
Fix error parsing to idt_int
**- How to verify it**
Test idt init done well
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/one-image.mk [hunk 1]
```diff
@@ -21,6 +21,7 @@ $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
                                $(ACCTON_AS7326_56X_PLATFORM_MODULE) \
                                $(ACCTON_AS7716_32XB_PLATFORM_MODULE) \
                                $(ACCTON_AS6712_32X_PLATFORM_MODULE) \
+                               $(ACCTON_AS7726_32X_PLATFORM_MODULE) \
```
Comment (lguohan) [inline platform/broadcom/one-image.mk]: please do not change the file mode, keep it as 644. Please also check all other such file mode changes make sure they are in right mode.

File: platform/broadcom/platform-modules-accton.mk [hunk 2]
```diff
@@ -8,6 +8,7 @@ ACCTON_AS7312_54X_PLATFORM_MODULE_VERSION = 1.1
 ACCTON_AS7326_56X_PLATFORM_MODULE_VERSION = 1.1
 ACCTON_AS7716_32XB_PLATFORM_MODULE_VERSION = 1.1
 ACCTON_AS6712_32X_PLATFORM_MODULE_VERSION = 1.1
+ACCTON_AS7726_32X_PLATFORM_MODULE_VERSION = 1.1
```
Comment (lguohan) [inline platform/broadcom/platform-modules-accton.mk]: 644

**Comment by lguohan** [comment]
please resolve conflict.

File: platform/broadcom/platform-modules-accton.mk (hunk 2 above)
Comment (jostar-yang) [reply platform/broadcom/platform-modules-accton.mk]: OK. I send "abdb2ff" commit to file mode to 644

File: platform/broadcom/one-image.mk (hunk 1 above)
Comment (jostar-yang) [reply platform/broadcom/one-image.mk]: OK. I send "abdb2ff" commit to file mode to 644

**Pull request merged**